### PR TITLE
📖 dead contributor and conduct links on cluster-api.sigs.k8s.io website

### LINKED ISSUE #1816: Page Not Found Dead Links on https://cluster-api.sigs.k8s.io/
**What steps did you take and what happened:**
Browsing https://cluster-api.sigs.k8s.io/ and clicking "contributor guide " hyperlink to https://cluster-api.sigs.k8s.io/CONTRIBUTING.html or "Kubernetes Code of Conduct" hyperlinked to https://cluster-api.sigs.k8s.io/code-of-conduct.html produced "Page Not Found".

**What did you expect to happen:**
Expected to be navigated to the top level CONTRIBUTING.md & code-of-conduct.md markdown based docs respectively in the browser.

**Anything else you would like to add:**

**Environment:**

-Browser: FF 70.0.1
-OS: Win 10

/kind bug

### PR DESCRIPTION
**Pull request by thesystemninjaneer**
**What this PR does / why we need it**:
Addresses two dead links on the README.md doc. Fixes #1816

Learning how to first time contribute code to this SIG by reading https://cluster-api.sigs.k8s.io/ and found a couple dead links in the documentation. Killing two birds with one stone by confirming my understanding of how to first time contribute by contributing a patch to those same docs! 🐦 🐦

There were a lot of docs about how to contribute so please let me know if I missed any steps!

### CODE REVIEW

**Comment by thesystemninjaneer** [comment]
I signed it

**Comment by vincepri** [comment]
@thesystemninjaneer The commit message shouldn't include "fixes #1816" text, can you remove it and push it again?

**Comment by thesystemninjaneer** [comment]
@vincepri Sure. I can remove that comment and also burn in the path to the markdown files in the repo.

File: README.md [hunk 1]
```diff
@@ -48,15 +48,15 @@ See also: our own [contributor guide] and the Kubernetes [community page].
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct].
 
 [community page]: https://kubernetes.io/community
-[Kubernetes Code of Conduct]: code-of-conduct.md
+[Kubernetes Code of Conduct]: https://github.com/kubernetes-sigs/cluster-api/blob/master/code-of-conduct.md
```
Comment (detiber) [inline README.md]: We should have a local code of conduct defined, do we no longer have this?

File: README.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ See also: our own [contributor guide] and the Kubernetes [community page].
...
 [notes]: https://docs.google.com/document/d/1Ys-DOR5UsgbMEeciuG0HOgDQc8kZsaWIWJeKJ1-UfbY/edit
 [recordings]: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
 [zoomMeeting]: https://zoom.us/j/861487554
 [implementerNotes]: https://docs.google.com/document/d/1IZ2-AZhe4r3CYiJuttyciS7bGZTTx4iMppcA8_Pr3xE/edit
 [providerZoomMeetingTues]: https://zoom.us/j/140808484
 [providerZoomMeetingWed]: https://zoom.us/j/424743530
 [issue tracker]: https://github.com/kubernetes-sigs/cluster-api/issues
-[contributor guide]: CONTRIBUTING.md
+[contributor guide]: https://github.com/kubernetes-sigs/cluster-api/blob/master/CONTRIBUTING.md
```
Comment (detiber) [inline README.md]: We should have a local contributing guide, did it get moved or removed?

**Comment by vincepri** [comment]
@detiber We have both defined locally, but that part of the readme is referenced in the quick start, and local references end up not working

**Comment by detiber** [comment]
> @detiber We have both defined locally, but that part of the readme is referenced in the quick start, and local references end up not working

This seems a bit error prone to hardcode to the `master` branch here...  Is this something that we should potentially translate as part of the build process somehow to reference the proper branch?

**Comment by vincepri** [comment]
We can probably remove those parts or overwrite the links in the quick start

**Comment by ncdc** [comment]
@vincepri @detiber did you all come to a resolution on your discussion here?

**Comment by detiber** [comment]
@ncdc I'm good with either of the approaches that @vincepri mentioned (excluding or overwriting at generation time).

**Comment by thesystemninjaneer** [comment]
@vincepri @detiber shall i push in changes like this https://github.com/kubernetes-sigs/cluster-api/pull/1832/files?

**Comment by detiber** [comment]
> @vincepri @detiber shall i push in changes like this https://github.com/kubernetes-sigs/cluster-api/pull/1832/files?

Ah, that could work as well.

**Comment by vincepri** [comment]
@thesystemninjaneer Are you able to make the changes @detiber suggested? Otherwise, we should open an issue to track the link generation work instead.

**Comment by thesystemninjaneer** [comment]
@vincepri yup. I'll work it this weekend. Are there any instructions to manually build the repo's mdocs? I'd like to test my changes locally.

**Comment by vincepri** [comment]
You can run `make serve-book` to run a local server

**Comment by thesystemninjaneer** [comment]
> 
> 
> @thesystemninjaneer Are you able to make the changes @detiber suggested? Otherwise, we should open an issue to track the link generation work instead.

Changes made. Tested locally with a [centos 8 container](https://github.com/thesystemninjaneer/cluster-api/blob/mdbook-container/docs/book/Dockerfile)

**Pull request merged**